### ISSUE TITLE
Creating ORGANICCHEMISTRY.py and adding recent pyqs to it in the format defined in readme

### ISSUE THREAD

**Issue by TriAay249** [TriAay249/JEEPYQREPLYBOT#1]

**Comment by sujaldev** [comment]
Hi, wouldn't it be better to add more questions once the hard-coded parts have been removed?

**Comment by TriAay249** [comment]
Perhaps, it is. I was suggesting this issue so that the bot doesn't start spamming repeated questions . However , removing hard-coded part seems better

**Comment by sujaldev** [comment]
I was thinking about running image recognition on the huge library of questions provided by NTA abhyas

**Comment by TriAay249** [comment]
Can you access the PYQs through the NTA Abhyas app? I thought it only had mocks.

**Comment by sujaldev** [comment]
Doesn't have PYQs but the mock questions and answers are available in a structured way. I'm looking into scraping PDF files, any suggestions on which PYQ books to scrape?

**Comment by TriAay249** [comment]
Arihant's Past 43 years Questions would be good . However , extending to other well known books like H.C. Verma 's , Solomons' would be fine too . But then ,  Copyright issues are a hassle.

**Comment by sujaldev** [comment]
I don't think we are going to run into any issues as long as the database is private and we are not selling anything...

**Comment by TriAay249** [comment]
Still, we got to abide by the rules mentioned in the book's foreword. I believe using NTA Abhyas' questions would be a much better and safer idea
